.github/workflows/version-code-and-release.yml

```diff
@@ -0,0 +1,59 @@
+name: Version Code Check and Draft Release
+
+on:
+  pull_request:
+    paths:
+      - 'lib/kql/**'
+      - 'lib/kibana/**'
+      - 'hunting/**'
+      - 'etc/**'
+      - 'pyproject.toml'
```

Comment:
Just checking for these paths, this will not catch changes to things like schemas, rule loader, etc. 
Should we include those in the versioning as well?

Reply:
etc/** should cover all schema changes 
<img width="237" alt="image" src="https://github.com/user-attachments/assets/748412db-c547-4acc-9e95-828f2ad52bf7">

Reply:
++ core repo should be added.